addons/website/static/src/js/content/menu.js

```diff
@@ -630,6 +630,22 @@ publicWidget.registry.hoverableDropdown = animations.Animation.extend({
             Dropdown.getOrCreateInstance(toggleEl).hide();
         }
     },
+    /**
+     * @private
+     * @param {Event} ev
+     * @param {String} method bootstrap dropdowns method name
+     */
+    _updateDropdownVisibility: function (ev, method) {
+        const { currentTarget } = ev;
+        if (config.device.size_class <= config.device.SIZES.SM ||
+            currentTarget.classList.contains('o_extra_menu_items')) {
+                return;
+        }
```

Comment:
Did not see the double indentation of `return` in my first review. Can you do this:
```suggestion
        if (config.device.size_class <= config.device.SIZES.SM
                || currentTarget.classList.contains('o_extra_menu_items')) {
            return;
        }
```
or this:
```suggestion
        const isMobile = config.device.size_class <= config.device.SIZES.SM;
        if (isMobile || currentTarget.classList.contains('o_extra_menu_items')) {
            return;
        }
```
or even this in this case would be fine:
```suggestion
        if (config.device.size_class <= config.device.SIZES.SM) {
            return;
        }
        if (currentTarget.classList.contains('o_extra_menu_items')) {
            return;
        }
```
:pray: